<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFile</class>
 <widget class="QDialog" name="CreateFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成文件选项</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成文件选项</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成项目：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Week">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>周计划/总结</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期一</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期二</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期三</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期四</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期五</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期六</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Day7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>星期日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_OK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成选项：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R_Study">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>只生成学习</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R_All">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成全部</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R_StudyPE">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学习＋体育</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
